gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonCallContext.java

```diff
@@ -326,7 +326,6 @@ public Duration getStreamIdleTimeout() {
     return streamIdleTimeout;
   }
 
-  @BetaApi("The surface for extra headers is not stable yet and may change in the future.")
```

Comment:
Issue to better support extraHeaders: https://github.com/googleapis/sdk-platform-java/issues/1752

Will be done in the future.

Reply:
If this BetaApi annotation is removed in gax-httpjson, should it also be removed in gax and gax-grpc?

Reply:
If the base class/interface is still `BetaApi`, we don't have to remove them, I would only remove `BetaApi` for httpjson specific classes.

Reply:
> If the base class/interface is still `BetaApi`, we don't have to remove them, I would only remove `BetaApi` for httpjson specific classes.

Just to clarify what you mean by httpjson specific classes. Do you mean:
1. Remove annotation for classes/ methods that exist ONLY in the httpjson module and no other gax module (`withExtraHeaders` would keep the BetaApi annotation)
2. Remove annotation only from the httpjson module (i.e. Remove in httpjson but keep in gax and gax-grpc)

Reply:
1 yes, 2 no. In addition, if a class does not extend from classes in other modules, we should remove `BetaApi` from it unless there is a reason not to(e.g. maybe the feature in grpc is also still in Beta).

Reply:
Gotcha. I'll check the files again and update.